src/components/Modal/index.tsx

```diff
@@ -115,7 +115,13 @@ export default function Modal({
       {fadeTransition.map(
         ({ item, key, props }) =>
           item && (
-            <StyledDialogOverlay key={key} style={props} onDismiss={onDismiss} initialFocusRef={initialFocusRef}>
+            <StyledDialogOverlay
+              key={key}
+              style={props}
+              onDismiss={onDismiss}
+              initialFocusRef={initialFocusRef}
+              dangerouslyBypassFocusLock
```

Comment:
The docs on this explicitly say that you should have some replacement focus lock if you enable this, but I don't see any replacement in this PR.